kuma/landing/views.py

```diff
@@ -12,9 +12,9 @@
 
 def home(request):
     """Home page."""
-    demos = (Submission.objects
-                       .all_sorted('upandcoming')
-                       .exclude(hidden=True))[:12]
+    demos = (Submission.objects.filter(featured=True)
+                               .exclude(hidden=True)
+                               .order_by('-modified').all()[:12])
```

Comment:
The `all()` at the end is not needed here.

As there is some infrastructure for sorting is already in place, I wonder if you want to add that type of filtered queryset to the `all_sorted` method as another option? Something like `Submission.objects.all_sorted('freshfeatured')` that then filters on featured and orders by modified as you have it here?